spec/artsy-eventstream/artsy/publisher_spec.rb

```diff
@@ -29,7 +29,7 @@
       allow(event).to receive(:verb).and_return(nil)
       expect { Artsy::EventService::Publisher.publish(topic: 'test', event: event) }.to raise_error 'Event missing topic or verb.'
     end
-    it 'calls publish on the exchange with proper data' do
+    it 'uses verb as routing key when calling publish on the exchange without passing routing_key' do
```

Comment:
I realize this is just extending the current behavior, but I'm wondering if it really makes sense to set the `routing_key` when not supplied going forward.

It seems like we're using verb in an overlapping way, and actually we could accomplish everything we need with _just_ routing keys. E.g., sending a message could specify `message.sent`, and consumers could subscribe to `message.*` or `message.sent` as desired. Does the addition of verb give us any new functionality?